### ISSUE TITLE
what kind of input files needed in there?

### ISSUE THREAD

**Issue by liamxg** [lauringlab/sarscov2_application#1]
<img width="765" alt="image" src="https://github.com/lauringlab/sarscov2_application/assets/46078579/b6c7ae15-dc27-40e0-ac47-d5c4ed414c4a">

**Comment by liamxg** [comment]
@julesjulesjules

**Comment by julesjulesjules** [comment]
this is still a work-in-progress / was part of a brainstorming idea of possible applications. the idea was be that you could upload a set of metadata about sequenced samples in the "Base File Input:" section, and the consensus FASTA file in the "FASTA File Input:" section, and it would auto process data through pangolin, nextclade, and then up into GISAID. The project has yet to move forward.

**Comment by liamxg** [comment]
@julesjulesjules meta data and sequence data?

**Comment by julesjulesjules** [comment]
Could you clarify/be more explicit about what you are asking?

**Comment by liamxg** [comment]
base file is for meta data, right?

**Comment by julesjulesjules** [comment]
Yes, if this app was fully functioning, the "Base File Input" section would be for uploading the metadata associated with the samples listed in the FASTA file that would be uploaded in the "FASTA File Input" section.
